qiskit/pulse/schedule.py

```diff
@@ -355,7 +355,7 @@ def draw(self, dt: float = 1, style: Optional['SchedStyle'] = None,
              scaling: float = None, channels_to_plot: Optional[List[Channel]] = None,
              plot_all: bool = False, plot_range: Optional[Tuple[float]] = None,
              interactive: bool = False, table: bool = True, label: bool = False,
-             framechange: bool = True):
+             framechange: bool = True, framechange_channels: bool = True):
```

Comment:
`framechange_channels` seems to indicate that the arg would be a list of channels rather than a bool. What about a name such as `show_framechange_channels`? I know it's a bit long; open to other ideas.

Reply:
thanks, it seems more suitable